src/extended_json.ts

```diff
@@ -347,116 +345,110 @@ function serializeDocument(doc: any, options: EJSONSerializeOptions) {
 }
 
 /**
- * EJSON parse / stringify API
- * @public
+ * Parse an Extended JSON string, constructing the JavaScript value or object described by that
+ * string.
+ *
+ * @example
+ * ```js
+ * const { EJSON } = require('bson');
+ * const text = '{ "int32": { "$numberInt": "10" } }';
+ *
+ * // prints { int32: { [String: '10'] _bsontype: 'Int32', value: '10' } }
+ * console.log(EJSON.parse(text, { relaxed: false }));
+ *
+ * // prints { int32: 10 }
+ * console.log(EJSON.parse(text));
+ * ```
  */
-// the namespace here is used to emulate `export * as EJSON from '...'`
-// which as of now (sept 2020) api-extractor does not support
-// eslint-disable-next-line @typescript-eslint/no-namespace
-export namespace EJSON {
```

Comment:
Sorry about this change looking large, removing the namespace in favor of the explicit export is favorable b/c the code generated in the bundling process is under our control rather than the tooling's

Reply:
Just comfirming there are no changes here just a diff because of the ns removal.

Reply:
There's at least the removal of SerializableTypes type which is not listed on the release note (I've just come into it during migration)

Reply:
Hey @KuSh, thanks for calling this out, and sorry about the surprise removal. I have filed a ticket for us to update the migration guide. At this time the workaround would be declaring the type in your code and casting, do you have a use case for that type that you could share (does it help with narrowing or something else)? I will capture what it provides so if we reintroduce it or something similar it remains useful. Thanks in advance!

https://jira.mongodb.org/browse/NODE-6302

Reply:
We were using those types for some function parameters and return type that were using parse and stringify internally. I have just changed them to unknown for now (as our policy forbids any) but this is clearly less strict than before and I'm not sure those fonctions accept more form of content so potentially more runtime errors than before